### PR TITLE
Fix grammer in Aryav's journey

### PR DESCRIPTION
**Pull request by KendallDoesCoding**
# Description

I have fixed "2" grammer errors in @aryav-v 's journey.

@aryav-v please review and our team (@collab-community/journey-book) will then merge!

# Reference
https://writer.com/grammar-checker/

### CODE REVIEW

**Comment by aryav-v** [comment]
@KendallDoesCoding Actually , the grammar fixes you have applied removed ```<br>``` tag at two places .

File: journeys/aryav-v.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # [Aryav Chaturvedi](https://github.com/aryav-v)'s Journey
 
-The spark of my interest into programming and development was a indirect consequence of HTML classes back in my school days .<br>
+The spark of my interest in programming and development was an indirect consequence of HTML classes back in my school days.
```
Comment (KendallDoesCoding) [inline journeys/aryav-v.md:3]:
```suggestion
The spark of my interest in programming and development was an indirect consequence of HTML classes back in my school days. <br>
```

add br back

File: journeys/aryav-v.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 
-Later, I began to explore various Operating Systems including Linux, hopping from one distro to another almost every week. Customizing my operating system
-helped me learn <i>Lua</i>, a programming language used in various configuration files . <br>
+Later, I began to explore various Operating Systems including Linux, hopping from one distro to another almost every week. Customizing my operating system helped me learn Lua, a programming language used in various configuration files.
```
Comment (KendallDoesCoding) [inline journeys/aryav-v.md:5]:
```suggestion
Later, I began to explore various Operating Systems including Linux, hopping from one distro to another almost every week. Customizing my operating system helped me learn Lua, a programming language used in various configuration files. <br>
```

add br back

**Comment by KendallDoesCoding** [comment]
> @KendallDoesCoding Actually , the grammar fixes you have applied removed `<br>` tag at two places .

Oops sorry. Didn't realize. Fixed, Please review now.

**Review by aryav-v — APPROVED** [review]
Its fixed now , thanks for your effort @KendallDoesCoding :)

**Review by vikasganiga05 — APPROVED** [review]
Looks good to me :+1:

**Comment by KendallDoesCoding** [comment]
Epic! No worries. It was all writer.com

**Pull request merged**